config/qt5_core.mpb

```diff
@@ -0,0 +1,56 @@
+project {
+  expand(QT5_INCDIR) {
+    $QT5_INCDIR
+    $(QT5_DIR)/include
+  }
+
+  requires += qt5
+  includes += $(QT5_INCDIR)
+  libpaths += $(QT5_DIR)/lib $(QT5_DIR)/bin
+  lit_libs += Qt5Core
+
+  verbatim (make, macros) {
+    CPPFLAGS += -fPIC
+  }
+
+  verbatim (gnuace, macros) {
+    CPPFLAGS += -fPIC
+  }
+
+  specific (prop:windows) {
+    // On Windows it's possible for Qt to include winsock.h through windows.h
+    // which will conflict with winsock2.h. This forces the preprocessor to skip
+    // winsock.h.
+    macros += _WINSOCKAPI_
```

Comment:
Let's see if another macro would be cleaner (something that's designed to be use that way).  This link has a suggestion: https://stackoverflow.com/questions/25532428/issue-with-winsock-when-adding-the-automoc-to-cmake-file

Reply:
Yeah, that works better. This one worked too but it gave redefinition warning because winsock2.h would also define `_WINSOCKAPI_`.